easybuild/easyblocks/r/r.py

```diff
@@ -55,19 +58,39 @@ def prepare_for_extensions(self):
         self.cfg['exts_filter'] = EXTS_FILTER_R_PACKAGES
 
     def configure_step(self):
-        """Configuration step, we set FC, F77 is already set by EasyBuild to the right compiler,
-        FC is used for Fortan90"""
-        environment.setvar('FC', self.toolchain.get_variable('F90'))
+        """Custom configuration for R."""
+
+        # define $BLAS_LIBS to build R correctly against BLAS/LAPACK library
+        # $LAPACK_LIBS should *not* be specified since that may lead to using generic LAPACK
+        # see https://github.com/easybuilders/easybuild-easyconfigs/issues/1435
+        env.setvar('BLAS_LIBS', os.getenv('LIBBLAS_MT'))
```

Comment:
I am using `$LIBBLAS_MT` here, while existing R easyconfigs use `$LIBBLAS`.

This matter mainly when using Intel MKL, i.e. multithreaded vs sequential, see also the discussion in #202.

We could/should make this configurable, and stick to linking with the sequential BLAS library by default?